### DISCUSSION TITLE
How can I have my `undefined` values treated as `null` and parsed to self-closing tag?

### DISCUSSION THREAD

**Discussion by taniasimpson** [NaturalIntelligence/fast-xml-parser#635] [Q&A]
Per changelog 3.9.11, null is handled and parsed to a self-closing tag. 

Right now if `undefined` is passed, we get `undefined` back in our xml which is causing issues. How can we handle `undefined` as `null` and have it replaced with a self-closing tag?? 

THANKS!

**Comment by amitguptagwl** [comment]
Hi @taniasimpson ,

Can you please share some sample code? It'll be helpful to understand the context of your question.
